File: app/src/main/java/com/capstone/project_niyakneyak/ui/main/listener/OnCheckedAlarmListener.java
package com.capstone.project_niyakneyak.ui.main.listener;

import com.capstone.project_niyakneyak.data.alarm_model.Alarm;

public interface OnCheckedAlarmListener {
    void OnItemClicked(Alarm alarm);
    void OnItemClicked(Alarm alarm, boolean isChecked);
}

File: app/src/main/java/com/capstone/project_niyakneyak/main/listener/OnCheckedAlarmListener.kt
package com.capstone.project_niyakneyak.main.listener

import com.capstone.project_niyakneyak.data.alarm_model.Alarm

interface OnCheckedAlarmListener {
    fun onItemClicked(alarm: Alarm)
    fun onItemClicked(medsID: Long, alarm: Alarm, isChecked: Boolean)
}